Lib/glyphs2ufo/torf.py

```diff
@@ -42,6 +43,25 @@
     '0.75,0.75,0.75,1',
     '0.25,0.25,0.25,1')
 
+WEIGHT_CODES = {
+    'Thin': 250,
+    'Light': 300,
+    'SemiLight': 350,
+    'DemiLight': 350,
+    '': 400,
+    'Regular': 400,
+    'Medium': 500,
+    'DemiBold': 600,
+    'SemiBold': 600,
+    'Bold': 700,
+    'ExtraBold': 800,
+    'Black': 900}
```

Comment:
In Google Fonts API, the Thin maps to CSS font-weight 100, and the fonts I manage with a "Thin" style also have a usWeightClass of 250

In Google Fonts API, the Light maps to CSS font-weight 300, and the fonts I manage with a "Light" style have a usWeightClass of 300. 

There is also an "ExtraLight" style, that maps to CSS font-weight 200, and usWeightClass 275.

SemiLight, DemiLight, DemiBold don't exist in the Fonts API. 

ExtraBold and Black are in the API and use the values here in CSS and usWeightClass.